### PR TITLE
Adjust print window for mobile

### PR DESCRIPTION
**Pull request by hblitza**
This uses custom css for mobile devices
- hiding the preview window and button
- initially uses auto size for panel width
- make use of `flex` in grid layout

Peek 2021-01-28 10-17

@terrestris/devs please note.

### CODE REVIEW

File: src/component/PrintPanel/PrintPanelV3.tsx [hunk 1]
```diff
@@ -645,7 +648,7 @@ export class PrintPanelV3 extends React.Component<PrintPanelV3Props, PrintPanelV
           </Col>
           {/* settings column */}
           <Col
-            span={12}
+            flex={2}
```
Comment (weskamm) [inline src/component/PrintPanel/PrintPanelV3.tsx:651]: cant we keep the old 50 / 50 split (setting this flex also to 1)?

**Comment by weskamm** [comment]
nice gif. just a question regarding split. else looking good

File: src/component/PrintPanel/PrintPanelV3.tsx (hunk 1 above)
Comment (hblitza) [reply src/component/PrintPanel/PrintPanelV3.tsx]: right, fixed.

**Comment by hblitza** [comment]
Thanks for review, merge and cleaning up ;)

**Pull request merged**